src/DynamoCoreWpf/UI/Themes/Modern/InPorts.xaml

```diff
@@ -200,6 +200,41 @@
                     CornerRadius="0,11,11,0"
                     IsHitTestVisible="True"
                     SnapsToDevicePixels="True" />
+
+            <!--  A border, to initiate the node autocomplete command -->
+            <Border x:Name="NodeAutoCompleteMarker"
+                    Grid.Column="0"
+                    Margin="-18,0,0,0"
+                    Cursor="Hand"
+                    CornerRadius="10"
+                    Height="20px"
+                    Width="20px"
+                    VerticalAlignment="Center"
+                    HorizontalAlignment="Left"
+                    Background="{StaticResource NodeTransientOverlayColor}"
+                    IsHitTestVisible="True"
+                    SnapsToDevicePixels="True"
+                    Visibility="{Binding NodeAutoCompleteMarkerVisible, Converter={StaticResource BooleanToVisibilityCollapsedConverter}}">
+
+                <Border.Effect>
+                    <DropShadowEffect BlurRadius="10"
+                                      ShadowDepth="0" 
+                                      Color="#D5BCF7"
```

Comment:
I think you can just reference from the resource dictionary without the inline definition of color

Reply:
Going to just remove the dropshadow as the binding isn't working. I also think it's a good idea to remove to reduce potential complexity